dist/modules/pager/pager.js

```diff
@@ -48,35 +50,38 @@ angular.module('c2fo.ui.components.pager', [])
                 '   </ul>',
                 '</nav>'
             ].join('').replace(/\s\s+/g, ''),
-            controller: function () {
-                var vm = this;
+            controller: 'PagerController',
+            controllerAs: 'pagerCtrl',
+            bindToController: true
+        };
+    }])
+    .controller('PagerController', [function () {
+        var vm = this;
 
-                vm.handlePageChange = handlePageChange;
-                vm.range = range;
+        vm.handlePageChange = handlePageChange;
+        vm.range = range;
 
-                function handlePageChange(pageNumber) {
-                    if (!pageNumber) {
-                        return false;
-                    }
+        function handlePageChange(pageNumber) {
+            if (!pageNumber) {
+                return false;
+            }
 
-                    vm.onPageChange()(pageNumber);
-                }
+            vm.onPageChange()(pageNumber);
+        }
 
-                function range(num, reverse) {
-                    var ret = [];
+        function range(num, reverse) {
```

Comment:
Instead of implementing our own `range` function we can just use [`_.range`](https://lodash.com/docs#range)